tests/voq/test_voq_fabric_status_all.py

```diff
@@ -0,0 +1,130 @@
+from tests.common.helpers.assertions import pytest_assert
+import logging
+import pytest
+import yaml
+
+logger = logging.getLogger(__name__)
+
+# This test only runs on t2 systems.
+pytestmark = [
+    pytest.mark.topology('t2')
+]
+
+# This test checks the fabric link status.
+# It loads the reference data for a linecard,
+# and runs the CLI command to get the link status
+# on the system and compares the output.
+
+
+def test_voq_fabric_link_status(duthosts):
```

Comment:
can we add a check for voq here or in conditional_marker to skip for no voq platforms?

Reply:
Thank you for your comments. I updated the check in conditional_marker.